specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2022-06-01/storagesync.json

```diff
@@ -4171,6 +4320,19 @@
           "readOnly": true,
           "type": "string",
           "description": "Resource Last Operation Name"
+        },
```

Comment:
I took a look at the documentation provided by ARM regarding the properties for MI: https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html?q=identityManagement

I don't see any mention there of identityType and identityId, it looks like everything should be contained inside the "identity" object instead (it already has "properties" that includes type)